### ISSUE TITLE
Tacotron-2 based on Parallel WaveNet?

### ISSUE THREAD

**Issue by peter05010402** [Rayhane-mamah/Tacotron-2#209]
Hello, I have trained wavenet model. But the speed of output wav file is low.
Does Tacotron-2 based on Parallel WaveNet, or not?

**Comment by veqtor** [comment]
Nope, no-one seems to have figured out how parallel wavenet should apply a shift and log scale function to a discretized mixture of logistics

**Comment by wordlessly** [comment]
So the paper on parallel wavenet doesn't explain this detail? Did the authors comment on this problem?

**Comment by veqtor** [comment]
Actually I think I know how now, you need to shift and scale a logistic noise (using 4 wavenets without residual connections) and minimize loss by evaluating it against a teacher wavenet conditioned on the same local condition as the IAF wavenet.
That is, you maximize the log prob of the iaf wavenet samples in the teacher distributions of gaussian mixtures

**Comment by wordlessly** [comment]
Does this mean we will see parallelization in the near future?
